### ISSUE TITLE
Exile Mod

### ISSUE THREAD

**Issue by jus61** [horbin/FuMS-HC-Server#3]
Do you supported Exile Mod

**Comment by ravmustang** [comment]
I am about to do my testing with fums, I'm not sure if horbin will or not....

**Comment by jus61** [comment]
would you share it with me

**Comment by ravmustang** [comment]
Fums is horbin beast I am more than willing to get with horbin and work with him, it's his mod, his credits.

I will share it with him and he can release his work if he wants with my edits. 
So far hc is perfect, everything sort of ok.
First changes are loot and equipment, then soldiers.....it goes deeper and tons of edits

**Comment by jus61** [comment]
ok

**Comment by ravmustang** [comment]
Yes I am also trying to write in zombies and demons mod into fums right now, kinda like he did for the jurassic mod.

**Comment by smithb895** [comment]
Can anyone confirm if this works with the latest version of ExileMod?  I know development has pretty much stopped on Exile, but I still run a server on Tanoa and was trying to get FuMS working.  I've setup the HC and as far as I can tell, everything is setup correctly, but the HC execution never seems to get to the FuMS part, I think it gets stuck at the following code block in the Exile.Tanoa/HC/init.sqf file:
`if (!isServer) then {
	waitUntil {!isNull player};
	waitUntil {player == player};
};
`
I've edited some of the FuMS code to skip over that block, and then it gets stuck printing the following lines in the HC logs:
`"<FuMS> HC_Init: <NULL-object> waiting on Server's Init Token"`
Judging from that line, it seems the player object is NULL, is that expected on a HC?  or is something wrong here?  

I appreciate any help or advice you can give.
